content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -84,7 +84,7 @@ is the [rewrite-target annotation](https://github.com/kubernetes/ingress-nginx/b
 Different [Ingress controllers](/docs/concepts/services-networking/ingress-controllers) support different annotations.
 Review the documentation for your choice of Ingress controller to learn which annotations are supported.
 
-The Ingress [spec](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status)
+The Ingress [spec](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec)
```

Comment:
The original link for `spec` is the api convention of spec not specific in Ingress.
This PR changes the intent of the link. If this was intentional than it's better to link `Ingress spec`
```suggestion
The [Ingress spec](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec)
```
This PR changes the intent of the link so that

Reply:
It's recommended to use a relative url like:

```
The [Ingress spec](/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec)
```

Reply:
> The original link for `spec` is the api convention of spec not specific in Ingress. This PR changes the intent of the link. If this was intentional than it's better to link `Ingress spec`
> 
> This PR changes the intent of the link so that

@reylejano Yes this PR changed the Intent of the link, it should link to [Ingress spec] not [spec].
I will update it accordingly.

Thanks!

Reply:
I have addressed the comment.
PTAL!